src/App.js

```diff
@@ -48,8 +52,8 @@ export default class App extends Component {
                 <Icons />
               </a>
             </Col>
-            <Col xs={8}>
-              
+            <Col>
```

Comment:
Needs to include a breakpoint.  Since first column uses breakpoint of 5, this should use xs={7}.